### PR TITLE
4.11.0 release

### PR DESCRIPTION
**Pull request by Blackbaud-AlexKingman**

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-datetime?_sp.spa=pr-4.11.0-release](https://developer.blackbaud.com/skyux-datetime?_sp.spa=pr-4.11.0-release)

**Review by blackbaud-paulah — DISMISSED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# 4.11.0 (2021-04-23)
+
+- Added the `startDateRequired` and `endDateRequired` input variables to the date range picker component. [#257](https://github.com/blackbaud/skyux-datetime/pull/257)
```
Comment (blackbaud-johnly) [inline CHANGELOG.md:3]: Is `variables` correct? Or should this just be `inputs`?

**Review by blackbaud-johnly — APPROVED** [review]
I added one question, but if `variables` is the right term, this LGTM.

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-datetime?_sp.spa=pr-4.11.0-release](https://developer.blackbaud.com/skyux-datetime?_sp.spa=pr-4.11.0-release)

File: CHANGELOG.md (hunk 1 above)
Comment (Blackbaud-AlexKingman) [reply CHANGELOG.md]: Updated.

**Review by blackbaud-johnly — APPROVED** [review]

**Pull request merged**